---
string[] drones_available
string message
bool success
